src/Gui/Dialogs/DlgAbout.cpp

```diff
@@ -580,6 +580,7 @@ void AboutDialog::copyToClipboard()
     QString point = QString::fromStdString(config["BuildVersionPoint"]);
     QString suffix = QString::fromStdString(config["BuildVersionSuffix"]);
     QString build = QString::fromStdString(config["BuildRevision"]);
+    QString disda = QString::fromStdString(config["BuildRevisionDate"]);
```

Comment:
I'm sorry but what does `disda` mean? Maybe `buildDate` would be a better name?
```suggestion
    QString buildDate = QString::fromStdString(config["BuildRevisionDate"]);
```